classes/blocks/class-framework.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace P4GBKS\Blocks;
+
+use P4\MasterTheme\Context;
+use P4\MasterTheme\Post;
+
+/**
+ * A block with the base template from master theme, inside which you can put post content.
+ */
+class Framework extends Base_Block {
+
+	public function __construct() {
+		register_block_type( 'planet4-blocks/framework', [
+			'editor_script'   => 'planet4-blocks',
+			'attributes'      => [],
+			'render_callback' => [ $this, 'render_it' ],
+		] );
+	}
+
+	public function prepare_data( $fields ): array {
+		return [];
+	}
+
+	public function render_it( $attributes, $content, $block ) {
```

Comment:
Name is not `render` as a quick fix to make it stop complaining the base classes method has different params.